packages/docs/pages/integrating-with-namada/sdk/shielded-transfers.mdx

```diff
@@ -56,4 +61,33 @@ async fn construct_transfer(sdk: impl Namada) {
       Err(e) => println!("Tx error: {:?}", e)
     }
 }
-```
\ No newline at end of file
+```
+
+### Frontend sustainability fee
+
+For transactions that move assets into or out of the shielded pool, the namada
+SDK provides the possibility to charge an optional fee to support the operations
+of the frontend provider. The specific transactions are:
+
+- `TxShieldingTransfer`
+- `TxUnshieldingTransfer`
+- `TxOsmosisSwap`
+- `TxIbcTransfer`
+- `GenIbcShieldingTransfer`
+
+<Callout type="warning">
+  For fully shielded osmosis swaps (from shieleded address to shielded address),
+  the operator should refrain from charging a fee.
```

Comment:
why should the operator refrain from charging a fee? I think a small justification is worthwhile here.